ui/pages/setup_backup_codes.tsx

```diff
@@ -104,18 +111,84 @@ const SetupBackupCodes: NextPage = () => {
     [flow, router],
   );
 
+  let hasDisableCodes = false;
   const lookupFlow = {
     ...flow,
     ui: {
       ...flow?.ui,
-      nodes: flow?.ui.nodes.filter(({ group }) => {
-        return group === "lookup_secret";
-      }),
+      nodes: flow?.ui.nodes
+        .filter(({ group }) => {
+          return group === "lookup_secret";
+        })
+        .map((node) => {
+          if (node.meta.label?.id === ORY_LABEL_BACKUP_CODE_CREATE) {
+            node.meta.label.text = "Create backup codes";
+            node.meta.label.context = {
```

Comment:
just a thought, is it possible to set react components in these ui node context? If that's valid, then we could move the backup code logic from `NodeInputSubmit` to this parent level?

Reply:
yes, that is a good idea, adopted to this with the last commit.